src/stylesheets/components/_footer.scss

```diff
@@ -272,3 +272,43 @@ li.usa-footer-primary-content {
     padding-left: 1em;
   }
 }
+
+.usa-social-link {
+  $hit-area: 4.5rem; // used for height and width
+  $left-align-adjust: 1.2rem; // relative left positioning
+  $blank-space: 1rem; // spacing between link hit areas
+
+  background-position: center center;
+  background-repeat: no-repeat;
+  display: inline-block;
+  height: $hit-area;
+  left: $left-align-adjust;
+  margin-left: $blank-space;
+  position: relative;
+  text-align: center;
+  width: $hit-area;
```

Comment:
I'm OK with keeping `$hit-area` for that reason, but would remove the rest. 

Can we add a more descriptive comment on it:

>Link hit target is 44 x 44 pixels following Apple iOS Human Interface Guidelines